dev_tools/scripts/add_license.sh

```diff
@@ -1,32 +1,79 @@
 #!/usr/bin/env bash
-shopt -s globstar
-shopt -s extglob
 
 THIS_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" &>/dev/null && pwd )
-GINKGO_ROOT_DIR="${THIS_DIR}/../.."
+# ${THIS_DIR} is expected to be: ${GINKGO_ROOT_DIR}/dev_tools/scripts
+
+# Use local paths, so there is less chance of a newline being in a path of a found file
+cd "${THIS_DIR}/../.."
```

Comment:
Technically, there should be no problem here because we expect this script to keep sitting in `dev_tools/scripts`, and even if it does move we personally would never use a space, newline, or anything in our file or directory names.

Reply:
That is true, but a user might have Ginkgo inside a weired path (previously, it was used as a global path), so there, it could be a problem.
I actually just wanted to minimize any possibility of something going wrong, which is why I chose to use the top level of Ginkgo as a working directory because I can be almost 100% certain that we don't have newlines in our folder structure (spaces should be fine with both scripts).